athenet/layers/inception.py

```diff
@@ -0,0 +1,82 @@
+"""Inception Layer."""
+
+import numpy as np
+
+import theano
+
+from athenet.layers import Layer, ConvolutionalLayer, ReLU, MaxPool, \
+    Concatenation
+
+
+class InceptionLayer(Layer):
```

Comment:
WeightedLayer would mean that InceptionLayer has W and b properties, which is not true. Only convolutional layers inside Inception have those.